src/rules/use-tokens/use-tokens.test.js

```diff
@@ -87,13 +87,13 @@ ruleTester.run('use-tokens', useTokens, {
       code: `
     const styles = StyleSheet.create({
       foo: {
-        color: 'rgba(0, 0, 0, 0.8)',
```

Comment:
a new token was added that uses this colour so we got a different error message saying that there's multiple colour tokens matching for this colour